iaas/20221116.md

```diff
@@ -0,0 +1,71 @@
+# 2022-11-16
+## Participants - use of github handle is preferred
+* @matfechner
+* @master-caster
+* @itrich
+* @ppkuschy
+* @jnull
+* @maliblatt
+* @fkr
+* @ra-beer
+* @maxwolfs
+* @garloff
+* @berendt
+* @o-otte
+* @lossack
+* @tibeer
+
+## Recurring items
+* Who is presenting in weekly community call and transforming etherpad to minutes in GitHub? @matfechner
+* Anything else to add to the agenda?
+* Fix date for hands-on session (2022-11-25 targeted). Who will take the lead?
+* OpenStack Image Manager
+* Housekeeping
+
+## Agenda
+
+### Flavor Standard -> PublicCloud SIG OpenInfra
+   @fkr will report from PublicCloud SIG
+
+   [publiccloud-sig-meeting](https://etherpad.opendev.org/p/publiccloud-sig-meeting)
+   - that progress is manageable
+   -  @garloff and @fkr will look over that and sync with SCS to see wether we can 
+    have a mutual standard between SCS and what OpenInfra propagates
+
+### local storage flavor PR
+
+SCS-Docs#150 https://github.com/SovereignCloudStack/Docs/pull/150
```

Comment:
The hyperlink was okay. I was only referring to the `[x] ==....==` annotation. See https://github.github.com/gfm/ for reference.